scene/2d/position_2d.cpp

```diff
@@ -66,6 +66,23 @@ void Position2D::_notification(int p_what) {
 	}
 }
 
+void Position2D::set_gizmo_length(float p_length) {
+	gizmo_length = p_length;
+	update();
+}
+
+float Position2D::get_gizmo_length() {
+	return gizmo_length;
+}
+
+void Position2D::_bind_methods() {
+
+	ClassDB::bind_method(D_METHOD("set_gizmo_length", "length"), &Position2D::set_gizmo_length);
```

Comment:
"length" sounds like something which only applies to a one dimensional object (a segment or a rod), while here we're talking about a square. Maybe here "size" would be better, even if kept as a float for both sides' lengths, or alternative "extents" for the half side length as used for rectangular shapes.

Reply:
Renamed to extents.